gusto_embedded/README.md

```diff
@@ -106,7 +106,7 @@ Generally, the SDK will work well with most IDEs out of the box. However, when u
 - [PyCharm Pydantic Plugin](https://docs.pydantic.dev/latest/integrations/pycharm/)
 <!-- End IDE Support [idesupport] -->
 
-<!-- Start SDK Example Usage [usage] -->
+<!-- No SDK Example Usage [usage] -->
 ## SDK Example Usage
```

Comment:
nit: Since we're maintaining this section can we rename this title to just `Example Usage`? I'd like us to drop the usage of "SDK" whenever possible to avoid confusions.